src/libraries/System.Threading/tests/MutexTests.cs

```diff
@@ -315,7 +316,7 @@ public void Ctor_TryCreateGlobalMutexTest_Uwp(bool currentUserOnly, bool current
                 Assert.Throws<UnauthorizedAccessException>(() => new Mutex(Guid.NewGuid().ToString("N"), options)));
         }
 
-        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported), nameof(PlatformDetection.HasAssemblyFiles))]
```

Comment:
This test does not appear to depend on assembly files. This condition does not look right.

Reply:
I think all failing named mutex tests should be disabled against https://github.com/dotnet/runtime/pull/114148/files#r2063936220

Reply:
Thanks, updated.